lib/i18n.rb

```diff
@@ -19,8 +19,8 @@ def setup_i18n_js
 
   insert_into_file 'app/views/layouts/application.html.erb', after: /<%= stylesheet_link_tag.*\n/ do
     <<~EOT
-      \t\t<%= javascript_include_tag 'i18n' %>
-      \t\t<%= javascript_include_tag 'translations/translations' %>
+          <%= javascript_include_tag 'i18n' %>
+          <%= javascript_include_tag 'translations/translations' %>
```

Comment:
Same comment here as the previous about the usage of lien break `\t`

Reply:
Same as above.